tests/wicked/startandstop/sut/t12_vlan_xml_config.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: VLAN - Create a VLAN from wicked XML files
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+
+use base 'wickedbase';
+use strict;
+use testapi;
+use network_utils qw(iface ifc_exists);
+
+sub run {
```

Comment:
what is the state of this HDD if you roll back in t11? So, it should be the same when t10 finished, right?

Reply:
we have hard linkage exclusively between t10 and t11 , t12 is totally isolated and have no link to things happening in t10. when t12 starts the state is same as for all other tests except t11 - state which we snapshot after before_test

Reply:
ok, thanks for clarification.